pkg/pdapi/pdapi.go

```diff
@@ -134,10 +140,26 @@ type pdClient struct {
 }
 
 // NewPDClient returns a new PDClient
-func NewPDClient(url string, timeout time.Duration) PDClient {
+func NewPDClient(url string, timeout time.Duration, tlsEnabled bool) PDClient {
+	httpClient := &http.Client{Timeout: timeout}
+	if tlsEnabled {
+		rootCAs, cert, err := httputil.ReadCerts()
+		if err != nil {
+			glog.Errorf("fail to load certs, fallback to plain connection, err: %s", err)
```

Comment:
Some problem with func `NewDefaultTiDBControl`

Reply:
The logic of `NewPDClient` is changed in #782 and does not have this issue anymore

Reply:
But func `NewDefaultTiDBControl `  still has this problem

Reply:
`NewDefaultTiDBControl` has been changed in the latest commit  7d9d03b